gulpfile.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var gulp = require('gulp');
+var gulpIstanbul = require('gulp-istanbul');
+var gulpMocha = require('gulp-mocha');
+
+gulp.task('test', function (cb) {
+  gulp.src(['dynamoDbQueueSource.js'])
+    .pipe(gulpIstanbul())
+    .pipe(gulpIstanbul.hookRequire())
+    .on('finish', function () {
+          gulp.src(['test/**/*.js'])
```

Comment:
2sp indentation. It's probably like this in wherever you copied it from as well. :)